terraform/modules/vault/variables.tf

```diff
@@ -110,6 +110,18 @@ variable "vault_config_dir" {
   default     = "/usr/local/etc/vault"
 }
 
+variable "vault_additional_config" {
+  type        = string
```

Comment:
I wish there was a way to make sure this input is valid hcl. Terraform now has [variable validation functions](https://www.terraform.io/docs/configuration/variables.html#custom-validation-rules) but I don't see an encoder for hcl. Alas!

Reply:
@lingrino the validation functions feature is really cool, I was not previously aware, thanks for sharing!